nrt/data/DynaSwing/cases.py

```diff
@@ -87,86 +87,126 @@
 
 test_cases.append((case_name, case_description, job_file, 5, standardReturnCodeType, standardReturnCode))
 
-###################################################
```

Comment:
Have you checked all the nrt ?

Reply:
I have just checked : 32 nrt failed
Checking back on master : 2 nrt failed

Main suspect : git ignore?

Reply:
They should work. I suggest you build dynawo again on your branch and check the nrt and then you push.